src/operon.ts

```diff
@@ -107,8 +109,15 @@ export class Operon {
       this.logger.debug("Using provided system database"); // XXX print the name or something
       this.systemDatabase = systemDatabase;
     } else {
-      this.logger.debug("Using Postgres system database");
-      this.systemDatabase = new PostgresSystemDatabase(this.config.poolConfig, this.config.system_database, this.logger);
+      if (this.config.systemDB === "postgres" || this.config.systemDB === undefined) {
+        this.logger.debug("Using FoundationDB system database");
```

Comment:
I think the debug message should be "Using Postgres"? Same for the debug message below, which should be "Using FoundationDB".